recipes/atsim-potentials/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "atsim-potentials" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/atsim.potentials/atsim.potentials-{{ version }}.tar.gz
+  sha256: 70082fc40b0ab7565a671c2d764fe3db08bc6ce45da44e1c1e8b77a65d1f7a23
+
+build:
+  number: 0
+  entry_points:
+    - potable=atsim.potentials.tools.potable:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - cexprtk >=0.3.4
+    - configparser
+    - funcsigs ==1.0.2  # [py<35]
```

Comment:
B/c we no longer build Python <3.5 we can probably remove this, the selector below, and make this noarch. I'll leave that to the feedstock since it is a recommendation and not a must.